article.js

```diff
@@ -0,0 +1,27 @@
+
+function CompleteArticle(dataObject) {
+  this.title = dataObject.title;
+  this.category = dataObject.category;
+  this.author = dataObject.author;
+  this.authorUrl = dataObject.authorUrl;
+  this.publishedOn = dataObject.publishedOn;
+  this.body = dataObject.body;
+}
+
+CompleteArticle.prototype.calculateDaysAgo = function (date) {
+  var milSecondsPerDay = 1000 * 3600 * 24;
+  var today = new Date();
+  return Math.floor((today-date) / milSecondsPerDay);
+}
+
+CompleteArticle.prototype.toHTML = function () {
+  var daysAgo = this.calculateDaysAgo(this.publishedOn);
+  var $newClone = $('article').filter(':first').clone();
+
+  $newClone.find(".articleTitle").text(this.title);
+  $newClone.find(".author").text("By " + this.author);
+  $newClone.find(".publishDate").text("Published about " + daysAgo + " days ago");
+  $newClone.find(".articleContent").html(fullArticle.body);
+
+  $("article:last").after($newClone);
+}
```

Comment:
Make sure your linter is functioning properly. I suspect it may be because of the wrong filename that I mentioned before. We want consistency here, so for example, the double quotes should be single quotes as per the linter spec. Most important is consistency, so on line 19 in this case, the single quotes are inconsistent (but as per the linter, we need all single quotes for this file). These little details are where tough-to-find bugs are born.